go/vt/throttler/throttler.go

```diff
@@ -224,6 +225,28 @@ func (t *Throttler) Throttle(threadID int) time.Duration {
 	return t.threadThrottlers[threadID].throttle(t.nowFunc())
 }
 
+// LastMaxLagNotIgnoredForTabletType returns the max of all the last replication lag values seen across all tablets of
+// the provided type, excluding ignored tablets.
+func (t *Throttler) LastMaxLagNotIgnoredForTabletType(tabletType topodata.TabletType) uint32 {
+	cache := t.maxReplicationLagModule.lagCacheByType(tabletType)
```

Comment:
Given my other comments, you could argue that lagCacheByType should be renamed to lagCache. If you did, I would agree :) Not necessary to include in this PR, but seems reasonable if you want to do it.

Reply:
There seems to already be a method with that name & different signature [here](https://github.com/vitessio/vitess/blob/3404baa17b47be5565fe48d0003ae63c3037411c/go/vt/throttler/max_replication_lag_module.go#L275), so I will leave the current name as it is for now.